docs/rfds/resume-session.mdx

```diff
@@ -0,0 +1,100 @@
+
+* it is exactly what we need
+* it is more straight-forward to implement for both clients and servers
+
+## What we propose to do about it
+
+> What are you proposing to improve the situation?
+
+Two possible options:
+
+* Add a "session/resume" command
+* Add an option to "session/load" which allows us to continue even if
+  it doesn't return the previous conversation
+
+In the rest of the document, I will refer to the solution as "session/resume",
+but it implies any of the solutions above.
+
+## Shiny future
+
+> How will things will play out once this feature exists?
+
+We will be able to resume existing conversations, providing a better user experience.
+
+## Implementation details and plan
+
+> Tell me more about your implementation. What is your detailed implementation plan?
+
+<!--
+    Use this section to add details that were not covered in the "What we propose to do about it" section and also include an implementation plan with phases.
+
+    Note: This section is OPTIONAL and NOT RECOMMENDED when RFDs are first opened. It can distract from the discussion of the problem.
+-->
+
+## Frequently asked questions
+
+> What questions have arisen over the course of authoring this document or during subsequent discussions?
+
+<!--
+    Keep this section up-to-date as discussion proceeds. The goal is to capture major points that came up on a PR or in a discussion forum -- and if they reoccur, to point people to the FAQ so that we can start the dialog from a more informed place.
+-->
+
+### What alternative approaches did you consider, and why did you settle on this one?
+
+The biggest question is if it makes sense to support both "session/load" and
+"session/resume". However, I do wonder: how portable "session/load" is in practice?
+
+When we start a new session over ACP, we introduce custom MCP tools and configuration.
+This means that, while we could use "session/load" to load our own chats and chats
+not started by us, I presume loading third-party chats would lead to a flawed user
+experience, as our tools would not be available. And depending on the ACP implementation,
+not even the capabilities would be respected.
+
+Therefore, I see functionality such as "session/load" and "session/list" being
+useful for those who want to provide a different interface for coding agents, without
+injecting custom functionality. Outside of that, the other benefit of "session/load"
+is that we could rely on the coding agent's own storage, but unfortunately that does
+not seem to be exposed at the moment. So if we have to emulate "session/load" at the
+ACP layer, we may know have chats stored in the coding agent, in ACP, and in our
+applications.
+
+If we can trivially implement "session/load" across agents, then I don't believe this
+RFD has weight. But assuming we don't, we may need to deal with these trade-offs:
+
+* If "session/resume" exists, coding agents may avoid implementing "session/load",
```

Comment:
It seems like a reasonable fallback. If an agent supports the others, it makes for a nicer experience, and then it is a matter of how much the client wants to handle fallbacks for resume I guess.